package.json

```diff
@@ -29,11 +30,13 @@
         "@hapi/code": "8.x.x",
         "@hapi/hapi": "20.x.x",
         "@hapi/lab": "24.x.x",
+        "@types/hapi__hapi": "^20.0.3",
         "node-forge": "0.10.x",
-        "node-rsa": "1.x.x"
+        "node-rsa": "1.x.x",
+        "typescript": "^4.1.3"
```

Comment:
I think we'd need to keep the version format we use i.e `20.x.x` for `@types/hapi__hapi` and `typescript` dependencies.

Reply:
Done @Nargonath 😉

Reply:
Alright, thanks 😃